### ISSUE TITLE
Navigation bars take up too much space

### ISSUE THREAD

**Issue by yannizhang2019** [BookStackApp/BookStack#3801]
### Attempted Debugging

- [X] I have read the debugging page

### Searched GitHub Issues

- [X] I have searched GitHub for the issue.

### Describe the Scenario

I recently migrated from the Bookstack version set up in 2018 to the current version, and notice visual changes. The redesign happened apparently in 2019 (https://github.com/BookStackApp/BookStack/pull/1153). Now the Details and Actions show to the right side of the page, and with the left side page navigation, they just take too much space (almost half of the page). Is there a way to preserve the old look and feel, or customize the page, perhaps the size of each sections, or a way to put the vertical menu bars to the upper? 
See the difference of the new page and old page (2018)

Page in current version, Details and Actions are at the right side-

<img width="1705" alt="image" src="https://user-images.githubusercontent.com/56080500/197422503-9db1a3a1-4bc3-45f9-bdcb-020508c348d8.png">

Page in 2018's version, Actions are at the top and Details are at the left, which give more space to display the page content.
<img width="1616" alt="image" src="https://user-images.githubusercontent.com/56080500/197422685-0c4d76d4-2c9a-4545-92a0-de52c4b8fa42.png">

### Exact BookStack Version

latest version

### Hosting Environment

Macbook

**Comment by ssddanbrown** [comment]
Hi @yannizhang2019,

>  which give more space to display the page content.

When I made the design changes, if I remember correctly, I did take care to preserve the page content width in most cases. 
The old design did give the impression of being wider, since the content container would stretch and appear part of the same "surface", due to the shared white background, as the content area itself.

> Is there a way to preserve the old look and feel, or customize the page, perhaps the size of each sections, or a way to put the vertical menu bars to the upper?

No, this is not possible via any built-in controls. We have a [visual theme system](https://github.com/BookStackApp/BookStack/blob/development/dev/docs/visual-theme-system.md) that allows overriding of any view BookStack uses internally, otherwise custom CSS could be used via the "Custom HTML Head Content" customization setting. Issue #1757 discusses tweaks that can allow the main content area to be wider.
